translib/yanglib_app.go

```diff
@@ -160,8 +161,8 @@ func (app *yanglibApp) processGet(dbs [db.MaxDB]*db.DB) (GetResponse, error) {
 	}
 
 	if err == nil {
-		resp.Payload, err = generateGetResponsePayload(
-			app.pathInfo.Path, app.ygotRoot, app.ygotTarget)
+		var root ygot.GoStruct = app.ygotRoot
```

Comment:
Wouldn't `root := app.ygotRoot` be enough?

Reply:
No.. `app.ygotRoot` is of `ygot.ValidatedGoStruct` type. Cannot cast directly to `*ygot.GoStruct`